cmd/importer/data.go

```diff
@@ -19,7 +19,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/cznic/mathutil"
```

Comment:
Can we remove the dependence `github.com/cznic/mathutil` from `go.mod` to avoid using it in our code base again?

Reply:
Seems not, we still need to depend on it in `util/mathutil`.😓